app/src/events.ts

```diff
@@ -1,8 +1,11 @@
 import mitt from 'mitt';
 
-const emitter = mitt();
+export const emitter = mitt();
 
-export default emitter;
 export enum Events {
 	upload = 'upload',
+	tabIdle = 'tab-idle',
+	tabActive = 'tab-active',
 }
+
+export default emitter;
```

Comment:
Sounds like a plan, but rather do that in a follow up PR as to not explode this PR by editing a lot of app files that aren't directly related to this change.

Reply:
Since we've adjusted the export as part of this PR, I think it's fine to also update the imports. We're talking about a manageable number of files:
```
src/utils/upload-file.ts
2:import emitter, { Events } from '@/events';

src/idle.ts
2:import emitter, { Events } from './events';

src/auth.ts
9:import emitter, { Events } from './events';

src/components/v-upload.vue
3:import emitter, { Events } from '@/events';

src/modules/files/routes/collection.vue
8:import emitter, { Events } from '@/events';
```

Reply:
Ah thought there may have been more but was probably confusing it with the api `emitter`. That seems managable, i'll update that 👌